pkg/command/cluster/clusterupdate/cmd_integration_api_test.go

```diff
@@ -60,6 +61,17 @@ func TestSctoolClusterUpdateIntegrationAPITest(t *testing.T) {
 				ForceNonSslSessionPort: true,
 			},
 		},
+		{
+			name: "update cluster, add labels",
+			args: []string{"cluster", "update", "--label", "k1=v1,k2=v2"},
+			expectedCluster: &models.Cluster{
+				Labels: map[string]string{
+					"k1": "v1",
+					"k2": "v2",
+				},
+				ForceTLSDisabled: true,
+			},
```

Comment:
Please change the initial state of the cluster lines 82-88 and add some label here, and change the args the way that one of the labels is removed.